Task5/css/main/tour/__old-price.css

```diff
@@ -0,0 +1,15 @@
+.tour__old-priсe {
+    visibility: hidden;
+}
+
+.sale-label .tour__old-priсe {
```

Comment:
Let's discuss. You should use modifiers for such cases according to the BEM rules.

Reply:
Renamed